python_modules/libraries/dagster-postgres/dagster_postgres/event_log/event_log.py

```diff
@@ -194,17 +194,18 @@ def store_asset_event(self, event):
 
         values = self._get_asset_entry_values(event, self.has_secondary_index(ASSET_KEY_INDEX_COLS))
         with self.index_connection() as conn:
-            conn.execute(
-                db.dialects.postgresql.insert(AssetKeyTable)
-                .values(
-                    asset_key=event.dagster_event.asset_key.to_string(),
-                    **values,
-                )
-                .on_conflict_do_update(
+            query = db.dialects.postgresql.insert(AssetKeyTable).values(
```

Comment:
I think we need the same fix in mysql and (maybe) in cloud.

Reply:
Thanks @prha for the catch. I've updated mysql to handle an empty dictionary of values (was having trouble adding a backcompat test, so just tested locally), let me know if you think we need to add a backcompat test.

Internal is okay, it handles an empty dictionary of values: https://github.com/dagster-io/internal/blob/14c4459385ddbf84e875b2be3c24fa2d88943d11/dagster-cloud/python_modules/ursula/ursula/storage/host_cloud/event_log_storage/storage.py#L227

Reply:
ideally, we would add a backcompat test for this... what was the issue you were running into?  lemme know if it's helpful to walk through...